### DISCUSSION TITLE
How to get italics for fira code like in the screenshots

### DISCUSSION THREAD

**Discussion by AmitGolden** [crivotz/nv-ide#7] [Q&A] [answered]
Hi,
How could I get the italic variant for fira code like in the screenshots?
From my understanding it has no italic variant

**Comment by crivotz** [comment, marked as answer]
This row made the difference in my case:
https://github.com/crivotz/nv-ide/blob/2ee7d1f4c6ffbfd30f463e6deea2b15cde494df4/lua/settings/color.lua#L6

I don't have hany type of firacode installed on my machine:
![Schermata del 2021-09-29 14-39-15](https://user-images.githubusercontent.com/3275600/135269958-6b9d5a9a-feca-4efa-948a-882a92c4cbda.png)

**Comment by AmitGolden** [comment]
And this works also while running neovim inside your terminal emulator? (because a gui like Neovide can do this on its own)

**Reply by crivotz** [reply]
Yes, i use neovim only from terminal. Here my [configuration](https://github.com/crivotz/st) for st
